sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/EventHubsConnectionStringBuilder.cs

```diff
@@ -287,6 +318,20 @@ void Validate()
                     string.Format("{0},{1}", SharedAccessKeyNameConfigName, SharedAccessKeyConfigName),
                     Resources.ArgumentInvalidCombination.FormatForUser(SharedAccessKeyNameConfigName, SharedAccessKeyConfigName));
             }
+
+            if (this.aadManagedIdentity.HasValue && hasSharedAccessSignature)
```

Comment:
>this.aadManagedIdentity.HasValue [](start = 16, length = 32)

Any chance if aadManagedIdentity == false?

Reply:
Key name is changed now.